apps/shopify/src/utils/base64.js

```diff
@@ -19,8 +19,8 @@ export const convertStringToBase64 = (str) => {
 export const convertProductToBase64 = (res) => {
   return {
     ...res,
-    id: convertStringToBase64(res.id),
-    product: res.product && { ...res.product, id: convertStringToBase64(res.product.id) },
+    id: convertStringToBase64(res?.id),
+    product: res?.product && { ...res?.product, id: convertStringToBase64(res?.product.id) },
```

Comment:
This adds tolerance to if the product/collection/variant were to be deleted when selected, it'll show up as product missing:
<img width="650" alt="image" src="https://user-images.githubusercontent.com/38636581/228939234-7d9d64eb-c1e6-45bd-a6c6-91911f415329.png">